### PR TITLE
Converts damage flags from string to defines, renames vague var

### PR DESCRIPTION
**Pull request by Tsar-Salat**
Ports:
- https://github.com/tgstation/tgstation/pull/53158/

## About The Pull Request
This PR renames all damage flags from a string to an actual define. This mostly is for easier coding, but also brings us more in line with TG for an eventual https://github.com/tgstation/tgstation/pull/71986 merge (ike liked it)

Also, I changed a var in projectile literally called "var/flag" regarding armor to be a little less vague. Its now var/armor_flag

## Why It's Good For The Game
Stuff like damage which is copied over and over again should really be in standardized defines. Putting it in strings makes it way too easy to call the wrong thing (did you know like 7 other things in the code use fucking "fire", very fun to manually snake around that when mass editing 200+ files)

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

worksa

weapons

</details>

## Changelog
:cl: RKz
code: changed damage flags into defines instead of strings
code: renamed a define in gun pointing, because it had FIRE as a define and made a conflict. Changed it to SHOOT
code: changed a vague projectile "var/flag"(wow so creative) to reflect what it actually does, "var/armor_flag"
/:cl:

### CODE REVIEW

File: code/datums/components/aiming.dm [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #define DROP_WEAPON "drop_weapon"
 #define DROP_TO_FLOOR "drop_to_floor"
 #define CANCEL "cancel"
-#define FIRE "fire"
+#define SHOOT "fire"
```
Comment (Tsar-Salat) [inline code/datums/components/aiming.dm]: Gun aiming component I had to rename

**Review by Penwin0 — CHANGES_REQUESTED** [review]
You've missed quite a few instances that haven't been converted to defines, too many for me to list out here. Please take another look.
I've only looked at the damage defines so far

File: code/datums/components/aiming.dm (hunk 1 above)
Comment (Penwin0) [reply code/datums/components/aiming.dm]: might as well change the string too while you are at it

Comment (Tsar-Salat) [reply code/datums/components/aiming.dm]: gotcha

**Comment by Tsar-Salat** [comment]
> You've missed quite a few instances that haven't been converted to defines

What?
I changed the all defines that were in the OG /tg/ pr, which were mostly armor lists or armor checks. I dont see any strings I could have missed.

**Comment by Tsar-Salat** [comment]
why the fuck is another branch pushing to this one

fuck off

**Pull request closed without merging**